src/Controller/CRUDController.php

```diff
@@ -78,11 +78,14 @@ class CRUDController implements ContainerAwareInterface
      */
     private $templateRegistry;
 
+    /**
+     * NEXT_MAJOR: We should not use this method for configuration, create a listener to call configureAdmin method.
+     */
     public function setContainer(?ContainerInterface $container = null)
     {
         $this->container = $container;
 
-        $this->configure();
+        $this->configure('sonata_deprecation_mute');
```

Comment:
why not call the new `configureAdmin` method here? then no need to mute the deprecation, or?

Reply:
In case someone override `$this->configure` maybe ?

Reply:
ah maybe people have overwritten the `configure` method :cry: forget it.